addok/http/base.py

```diff
@@ -82,6 +83,7 @@ def to_geojson(self, req, resp, results, query=None, filters=None,
             "features": [r.format() for r in results],
             "attribution": config.ATTRIBUTION,
             "licence": config.LICENCE,
+            "addok": VERSION,
```

Comment:
"version" is reserved for schema version (GeoCodeJSON version in theory), so I used this key instead. Any better idea?